tests/ui/test_ocs_odf_upgrade_ui.py

```diff
@@ -0,0 +1,84 @@
+from ocs_ci.ocs.ui.base_ui import logger
+from ocs_ci.ocs.ui.views import locators, ODF_OPERATOR
+from ocs_ci.utility.utils import get_ocp_version
+from ocs_ci.framework import config
+from ocs_ci.ocs.ocs_upgrade import OCSUpgrade
+from ocs_ci.ocs.ui.deployment_ui import DeploymentUI
+from ocs_ci.ocs.ui.validation_ui import ValidationUI
+from ocs_ci.ocs.resources.storage_cluster import ocs_install_verification
+from ocs_ci.framework.testlib import ManageTest, skipif_ocs_version, skipif_ocp_version
+
+
+@skipif_ocp_version("<4.9")
+@skipif_ocs_version("<4.8")
+class TestOcsOdfUpgrade(ManageTest):
+
+    ocp_version = get_ocp_version()
+    dep_loc = locators[ocp_version]["deployment"]
+    validation_loc = locators[ocp_version]["validation"]
+    operator = ODF_OPERATOR
+
+    def test_ocs_odf_upgrade(self, setup_ui):
```

Comment:
I would call such upgrade from our regular upgrade test in the case some config value to do UI upgrade is provided to ocs-ci and do UI upgrade instead of CLI as we have now.

We have upgrade job sets to call upgrade ocs-ci run with such markers:
`pre_upgrade or pre_ocs_upgrade or ocs_upgrade or post_ocs_upgrade or post_upgrade`

So the logic still has to be same in order to keep also specific order of the tests. O this logic here should not be defined as test but as regular class/methods and inside our regular upgrade test to decide if do CLI upgrade or the new UI. Then the same verification like we do for checking images and so on has to be invoked as well.